src/EA.Weee.Web/Areas/Admin/Views/Producers/_detailsResults.cshtml

```diff
@@ -78,7 +78,7 @@
                     @if (@scheme.CompanyNumber != null)
                     {
                         <tr class="govuk-table__row">
-                            <th class="govuk-table__header" scope="row">Company registration number</th>
+                            <th class="govuk-table__header" scope="row">company registration number</th>
```

Comment:
Dont think this should change as its heading.

Reply:
restored